Updated autopush

merspieler opened the pull request:
Signed-off-by: merspieler <merspieler@airmail.cc>

<!-- Give a brief summary of your changes in the title. -->

### Description of Changes
Updated autopush to version 2.0.1 plus view fix for fg version >2019.2

### Screenshots (optional)

### Bugs fixed (if any)
<!-- If you fixed any bugs, describe them here. State issue number if applicable. -->

### Checklist:
<!-- [ ] = Unchecked, [x] = Checked. -->
* [x] My changes follow the Contributing Guidelines. <!-- See CONTRIBUTING.md to verify. -->
* [ ] My changes implement realistic features. <!-- Only aircraft changes require this. -->
* [ ] Please have a main Developer test my changes before merging. <!-- We will always briefly test, but if it needs a "full" test, please check). -->
* [x] My changes are ready for merging. <!-- Uncheck if you want to decide when to merge. -->

legoboyvdlp commented:
Fantastic, I will check this right away 😃

legoboyvdlp commented:
` Nasal runtime error: setprop() passed a NaN
  381.06 [ALRT]:nasal        at C:/Users/redpa/Documents/FlightGear/Custom Aircraft/A320-family/Nasal/autopush_driver.nas, line 87
  381.06 [ALRT]:nasal        called from: C:/Users/redpa/Documents/FlightGear/Custom Aircraft/A320-family/Nasal/autopush_driver.nas, line 90`

legoboyvdlp commented:
Just checking - this is the very latest version on gitlab, right?

merspieler commented:
Yes, it's from HEAD

legoboyvdlp commented:
Simply load autopush and pushback?

legoboyvdlp commented:
mimino:

If steering is NaN I bet the settings snippet wasn't updated along
 
legoboyvdlpToday at 7:10 PM
yes
autopush branch has the new one (afaik) (edited)
 
miminoToday at 7:34 PM
no it doesn't
copy latest autopush nasals into it and see
but latest head, not latest release
because -next fixes not yet got released

merspieler commented:
I was able to reproduce the error... 1/10 tries.

I have no clue whats happening there. Most of the time I don't get anything.

One minor mistake I've noticed was that I've added the same prop twice but that should not have been the cause.

I've moved the autopush config to it's own file to make sure to use all the right props as well as making diffs against the template easier.

legoboyvdlp commented:
Ok, I can no longer reproduce, and it definitely appears to be latest autopush, so I will accept and merge.
